content/en/metrics/guide/custom_metrics_governance.md

```diff
@@ -0,0 +1,216 @@
+
+Metrics without Limits™ provides you with OOTB insights to assess the value of your actively queried metrics with [Metrics Related Assets][15]. A metrics related asset refers to any Datadog asset, such as a dashboard, notebook, monitor, or SLO that queries a particular metric. Use related asset popularity and quantity to evaluate metric utility within your organization, enabling data-driven decisions. Gain a better understanding of how your team can use existing metrics to get more value from your observability spend. 
+
+{{< img src="metrics/volume/related_assets.png" alt="Metric detail side panel showing the Related Assets section. The example metric is applied to one dashboard" style="width:100%;" >}}
+
+To view a metric's related assets:
+1. Click on the metric name to open its details side panel.
+1. Scroll down to the section of the side panel titled **Related Assets**.
+1. Click the dropdown button to view the type of related asset you are interested in (dashboards, monitors, notebooks, SLOs). You can use the search bar to validate specific assets. 
+
+## Monitoring and prevention
+
+After you have an understanding of your account's metric usage and the teams responsible for those costs, build monitors to alert you when your cardinality exceeds a certain threshold. Get alerts for overages and spikes in custom metric usage, so you can stay ahead of your monthly bill. 
+
+Datadog offers OOTB metrics that measures [estimated custom metrics usage][15]. You can use these metrics in your dashboard visualizations and monitor alerts. 
+
+| Usage Type                    | Metric                                   | Description |
+|-------------------------------|------------------------------------------| ----------- |
+| Indexed Custom Metrics        | `datadog.estimated_usage.metrics.custom`, `datadog.estimated_usage.metrics.custom.by_metric` | Unique indexed Custom Metrics seen in the last hour. |
+| Ingested Custom Metrics       | `datadog.estimated_usage.metrics.custom.ingested`, `datadog.estimated_usage.metrics.custom.ingested.by_metric` | Unique ingested Custom Metrics seen in the last hour. |
+
+### Monitor overages and suddent spikes in custom metric usage
+
+{{< img src="metrics/guide/custom_metrics_governance/estimated_usage_metric_monitor.png" alt="Example metric monitor configuration with the estimated usage metric and a threshold of 700,000" style="width:80%;" >}}
+
+You can see an analysis of your realtime estimated custom metrics usage by metric name with either the dashboard timeseries widget, or a metric monitor. Use the `datadog.estimated_usage.metrics.custom` metric to build a monitor so you can always have up-to-date visibility into each of your metric names' volumes. 
+
+### Additional monitoring use cases
+
```

Comment:
```suggestion
Once you've received an alert, use the Metrics Volume Management page to inspect any spiking metrics' tag keys and use Metrics without Limits™ to immediately drop any anomalous tag keys that are causing your metric to spike. This will ensure you can immediately resolve any unintentional billing spikes. 
```